assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -84,17 +91,19 @@ function TextPane(props) {
       )}
 
       <Section title={__('Presets', 'web-stories')}>
-        {PRESETS.map(({ title, element }, i) => (
-          <FontPreview
-            key={
-              /* eslint-disable-next-line react/no-array-index-key */
-              i
-            }
-            title={title}
-            element={element}
-            onClick={() => onClick(title, element)}
-          />
-        ))}
+        <GridContainer>
+          {PRESETS.map(({ title, element }, i) => (
+            <FontPreview
+              key={
+                /* eslint-disable-next-line react/no-array-index-key */
+                i
+              }
```

Comment:
I'm fine with this as is, but the title should also be a valid id if we don't want to disable eslint. don't think it has any impact on the actual functionality tho lol